src/ipyradiant/basic_tools/selection_widget.py

```diff
@@ -0,0 +1,99 @@
+import traitlets as trt
+
+import ipywidgets as ipyw
+
+
+class PredicateSelectionWidget(ipyw.HBox):

```

Comment:
Can we generalize this whole widget please? We can remove the extra buttons (like `collapse_literals`) and simplify the example to just a `MultiPanelSelectWidget` where there is a list of `available things` and a list of `selected things`.

Reply:
We can add buttons onto the widget in a future implementation.